.github/workflows/publish-dpp-verification-wallet-docker-image.yml

```diff
@@ -0,0 +1,161 @@
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ "main", "develop", "feature/simple-wallet" ]
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Branch to use"
+        required: true
+        default: 'main'
+        type: string
+
+env:
+  IMAGE_NAME: 'digital-product-pass-verification-wallet'
+  REGISTRY: 'ghcr.io'
+  IMAGE_NAMESPACE: 'tractusx'
+  COMMIT_SHA: ${{ github.sha }}
+  REPO_URL: ${{ github.server_url }}/${{ github.repository }}
+
+jobs:
+  build-on-docker:
+    if: github.repository == 'eclipse-tractusx/digital-product-pass'
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      # Build actions for docker hub registry
+      - name: Docker meta
+        id: meta-for-dockerhub
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}
+            type=semver,pattern={{major}}.{{minor}}
+
+      # Login against docker registry
+      # https://github.com/docker/login-action
+      - name: Log into a Docker registry
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      # Build and push Docker image with Buildx
+      # https://github.com/docker/build-push-action
+      - name: Build and push frontend
+        id: build-and-push-dpp-wallet-dockerhub
+        uses: docker/build-push-action@v5
+        with:
+          context: dpp-verification/simple-wallet
+          push: true
+          tags: ${{ steps.meta-for-dockerhub.outputs.tags }}, ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}:latest
+          labels: ${{ steps.meta-for-dockerhub.outputs.labels }}
+          build-args: |
+            REPO_COMMIT_ID=${{ env.COMMIT_SHA }}
+            REPO_ENDPOINT_URL=${{ env.REPO_URL }}
+
+
+      # https://github.com/peter-evans/dockerhub-description
+      # Important step to push image description to DockerHub 
+      - name: Update Docker Hub description
+        uses: peter-evans/dockerhub-description@v4
+        with:
+        # readme-filepath defaults to toplevel README.md, Only necessary if you have a dedicated file with your 'Notice for docker images'   
+          readme-filepath: docs/notice.md
+          username: ${{ secrets.DOCKER_HUB_USER }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+          repository: ${{ env.IMAGE_NAMESPACE }}/${{ env.IMAGE_NAME }}
+
+  build-on-ghcr:
```

Comment:
remove this block of ghcr, it is no more used and should not be referenced anywhere